tailwind.config.js

```diff
@@ -0,0 +1,169 @@
+module.exports = {
+  theme: {
+    backgroundOpacity: {
+      0: '0',
+      20: '0.2',
+      30: '0.3',
+      40: '0.4',
+      80: '0.8',
+      100: '1'
+    },
+    borderColor: theme => ({
+      1: theme("colors.border.1"),
+      2: theme("colors.border.2"),
+      3: theme("colors.border.3"),
+      4: theme("colors.border.4"),
+      5: theme("colors.border.5"),
+    }),
+    borderWidth: {
+      0: "0",
+      1: "1px",
+      3: "3px"
+    },
+    borderRadius: {
+      none: '0',
+      1: "var(--calcite-border-radius)",
+      2: "30px",
+      3: "50px",
+      half: '50%',
+      full: '100%',
```

Comment:
`2`, `3`, and `half` seem like they would be (or should be?) rarely used; maybe :axe: these?

Reply:
Yeah I was kind of.reverse-engineering some values for this based on border-radius values currently used in the component set. Interested in @macandcheese 's input on what these should be. I think floating buttons are the only large border-radius thing?

Reply:
Agreed, even half seems unlikely to be used. If we were to ever add config options for border radius, 0/1/2 could be reasonable, but as it stands we use `0px` for 1, and `3px` for 2 - as a config it might be someone uses `3px` for 1 and `9px` for 2...

Reply:
OK, I just removed the extras for now, we can revisit in the future as we start to use this stuff.